run_unittests.py

```diff
@@ -1008,6 +1008,42 @@ def test_compiler_options_documented(self):
                 self.assertIn(opt, md)
         self.assertNotIn('b_unknown', md)
 
+    def test_builtin_options_documented(self):
+        '''
+        Test that universal options and base options are documented in
+        Builtin-Options.md.
+        '''
+        md = None
+        with open('docs/markdown/Builtin-options.md') as f:
+            md = f.read()
+        self.assertIsNotNone(md)
+
+        found_entries = set()
+        sections = list(re.finditer(r"^## (.+)$", md, re.MULTILINE)) + [None]
+
+        for s1, s2 in zip(sections[:], sections[1:]):
+            if s1.group(1) == "Universal options":
+                # Extract the content for this section
+                end = s2.start() if s2 != None else len(md)
```

Comment:
__[Flake8]__ [E711] comparison to None should be 'if cond is not None:' ([link](https://sider.review/gh/repos/19784232/pulls/5233#issue-3971914))

<sub>posted by Sider</sub>